dashboard/alerts/alert.py

```diff
@@ -87,29 +119,41 @@ def saveAlert(self):
         """
         try:
             # Open config file to get path
-            config_file = open('/etc/capstone-ids/config.json', 'r')
-            config_data = json.load(config_file)
-            path = config_data["alert_collection_path"]
-
-            # Default path
-            if path == "" or path is None:
-                path = "/etc/capstone-ids/alert_collection.json"
+            try:
+                config_file = open('/etc/tinyHIPPO/config.json', 'r')
+                config_data = json.load(config_file)
+                path = config_data["alert_collection_path"]
+            except:
+                path = "/etc/tinyHIPPO/alert_collection.json"
```

Comment:
Two points here
1. You no longer need to read from the config if you import the config object
2. Did you update the config on master to reflect this new value that you need?